### PR TITLE
.*: fix revive package-comments lint issues

### PR DESCRIPTION
**Pull request by purnesh42H**
Addresses: #7444 

RELEASE NOTES: None

### CODE REVIEW

File: examples/features/csm_observability/client/main.go [hunk 1]
```diff
@@ -16,6 +16,7 @@
  *
  */
 
+// Binary client is an example client.
```
Comment (dfawley) [inline examples/features/csm_observability/client/main.go:19]: Can you include the example's purpose/name here too?  E.g.

`// Binary client is an example client for CSM observability.`

Comment (purnesh42H) [reply examples/features/csm_observability/client/main.go]: Added. Should we change other examples as well? I just followed the other examples and they didn't have names

Comment (dfawley) [reply examples/features/csm_observability/client/main.go]: The goal is not to increase the scope of this work.  I'm fine with leaving existing things alone -- unless you _want_ to go change them, in which case I will approve.

In general, though, we should _never_ use "some other thing we have does X" as an excuse to do "X", if we know "X" is not optimal.

File: examples/features/customloadbalancer/client/main.go [hunk 2]
```diff
@@ -16,6 +16,7 @@
  *
  */
 
+// Binary client is an example client for custom load balancer.
```
Comment (dfawley) [inline examples/features/customloadbalancer/client/main.go:19]: This is not proper English.  "for _a_ custom load balancer" or "a client for the custom load balancer example", etc.

**Review by dfawley — APPROVED** [review]
One small nit, otherwise LGTM.  Thanks!

File: examples/features/customloadbalancer/client/main.go (hunk 2 above)
Comment (purnesh42H) [reply examples/features/customloadbalancer/client/main.go]: Thanks. Done

File: examples/features/csm_observability/client/main.go (hunk 1 above)
Comment (purnesh42H) [reply examples/features/csm_observability/client/main.go]: I agree. I was just not sure if it was intentional to leave out the names. I will open a small issue to improve the example documentation

**Pull request merged**